src/Saas.Admin/readme.md

```diff
@@ -8,70 +8,74 @@ The SaaS Admin Service is an API that is reponsible for tenant management operat
 
 3. Saas.Admin.Service.Tests - Unit tests for the service
 
-## Project Overview
+## 1. Module Overview
 
-For more information on this module, please see the admin service page on our documentation site [here](https://azure.github.io/azure-saas/components/admin-service/).
+This project hosts a service api which serves as a gateway to administrate the SaaS ecosystem of Tenants. It is fully self-contained such that it includes complete copies of all necessary classes for operation. Since it contains no direct references to the other projects, it can be extracted to launch in isolation insofar as expected substitutes are configured.
```

Comment:
```suggestion
This project hosts a service api which serves as a gateway to administrate the SaaS ecosystem of Tenants. It is fully self-contained such that it includes complete copies of all necessary classes for operation. Since it contains no direct references to the other projects, it can be extracted to launch in isolation. However, keep in mind that some functionality within the API does have [dependencies](https://azure.github.io/azure-saas/components/admin-service/#dependencies) on other services.
```